site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -23,7 +23,10 @@ This page contains the APIs that are used to manage Links that establish a relat
 This API is used to create a link between a FusionAuth user and a user in a 3rd party identity provider. This API may be useful when you already know the unique Id of a user in a 3rd party identity provider and the corresponding FusionAuth user.
 
 === Request
-
+[NOTE.since]
+====
+This API endpoint has been available since 1.43.0.
+====
```

Comment:
Can we use consistent wording here? See other APIs with this type of notice. 

Example:

```

[NOTE.since]
====
Available Since Version 1.43.0
====
```

Since this API request isn't new - should this just go after `==== Request Body`? or delete it and just add the new fields to the existing doc with a since flag, and mark old ones deprecated? 

@mooreds any preference?

Reply:
Consistency is good. 

My concern with adding new fields to the existing done and marking the old ones as deprecated is that a user could mix and match the old and new fields in a way that would be kinda weird. 

```
{
  "identityProviderLink" : {
    "displayName" : "richard@piedpiper.com",
  },
  "identityProviderId": "f50dbb83-4cc2-4e5e-aece-9efe068bddd9",
  "identityProviderUserId": "42",
  "userId": "fbf0e652-b2b5-45eb-b9c9-d4640889740b"
}
```

Would that even work? Or would it fail validation?

Reply:
I think it would work -would have to test - but it could be the result is un-predictable, it would depend upon the order that Jackson processes the request. Good point though - let's keep them separate.